### PR TITLE
transaction repository testing

### PR DESCRIPTION
**Pull request by alex-arugon**

### CODE REVIEW

File: core/repository/transaction/pg.ts [hunk 1]
```diff
@@ -56,8 +56,9 @@ class TransactionPGRepo implements ITransactionPGRepository<IAsset> {
         });
     }
 
-    async deleteById(trsId: string | Array<string>): Promise<void> {
-        await db.none(queries.deleteByIds, [trsId]);
+    async deleteById(trsId: string | Array<string>): Promise<Array<string>> {
+        const response = await db.manyOrNone(queries.deleteByIds, [trsId]);
```
Comment (axelsheva) [inline core/repository/transaction/pg.ts]: Will it works correctly when trsId will be string and array of string?

File: core/repository/queries/transaction.ts [hunk 2]
```diff
@@ -2,6 +2,6 @@
 export default {
     getByBlockIds: 'SELECT * FROM trs WHERE block_id IN ($1:csv)',
     getById: 'SELECT * FROM trs WHERE id = ${id}',
-    getTransactionBatch: 'SELECT * FROM trs ORDER BY created_at ASC LIMIT ${limit} OFFSET ${offset}',
```
Comment (axelsheva) [inline core/repository/queries/transaction.ts]: @Verbalist is it correct?

File: core/repository/transaction/pg.ts (hunk 1 above)
Comment (alex-arugon) [reply core/repository/transaction/pg.ts]: Tests cover both variants

File: core/repository/queries/transaction.ts (hunk 2 above)
Comment (Verbalist) [reply core/repository/queries/transaction.ts]: nope

**Pull request merged**